modules/elasticsearch/build.gradle

```diff
@@ -0,0 +1,20 @@
+description = "TestContainers :: elasticsearch"
+
+dependencies {
+    compile project(':testcontainers')
+    testCompile "org.elasticsearch.client:elasticsearch-rest-client:${elasticsearchVersion}"
+}
+
+
+processResources {
```

Comment:
Could you please explain why we need it?

Reply:
Do you mean as a comment within the Gradle file?
Or do you need an explanation here of why this is needed?

Reply:
Here, because that's something unusual for Testcontainers and I have some doubts about the need for it :)

Reply:
I like having properties defined in a single place to make upgrading versions super easily.
So I defined in `gradle.properties` file:

```
# The default image version we will use
elasticsearchVersion=6.3.2
# For integration tests we try also an older version
elasticsearchOldVersion=5.6.10
```

Using `processResources` and `processTestResources` helps to replace the values that are defined in:

* `src/main/resources/elasticsearch-default.properties`
* `src/test/resources/elasticsearch-version.properties`

Reply:
We usually store versions as constants in the container class.
One of the downsides of `processResources` is that the tests can't be started from IDE if "delegate to Gradle" is not selected

Could you please remove the properties and use constants, for the sake of consistency & friendliness to IDE users? :)

Reply:
I can... But this will have downsides when playing with plugins as for some tests, I must download locally a plugin which version is exactly the same as the elasticsearch one.

I mean that will make elasticsearch version more complicated as you'll have to change the version number in multiple places.
I can do it but please confirm that this is what you want.

Reply:
I think that can be solved in imperative way without using the properties
Also, some users are running their own registries and using their own images, for them `elasticsearchVersion` would not work :)

Let's think how to solve the situation with the plugins in the plugins PR, I hope it's ok to you :)

Reply:
I just pushed a new change.